internal/commands/app/create_inputs_test.go

```diff
@@ -248,6 +248,12 @@ func TestAppCreateInputsResolveDirectory(t *testing.T) {
 		assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{Realm: rc}))
 
 		existingDir := filepath.Join(profile.WorkingDirectory, existingApp.Name)
+		assert.Nil(t, ioutil.WriteFile(
```

Comment:
these are odd existing tests b/c it seems like we're using the command `Handler` in order to setup the test env for the comamnd's `inputs.resolveLocalPath` to fail

so I guess my question is: what broke here?  I would guess that the `CommandCreate.Handler` would still be creating the config version like you're writing it here?

Reply:
you're right. the command `Handler` was creating the config file. The issue was there was no config version being written to it so `local.FindApp` couldn't recognize it. 
Opted for simply adding the ConfigVersion to the createInputs instead of writing the config file.